sci-libs/nibabel/nibabel-1.3.0.ebuild

```diff
@@ -10,7 +10,7 @@ inherit distutils-r1
 
 DESCRIPTION="Access a cacophony of neuro-imaging file formats"
 HOMEPAGE="http://nipy.org/nibabel/"
-SRC_URI="https://github.com/nipy/nibabel/archive/${PV}.zip"
+SRC_URI="https://github.com/nipy/nibabel/archive/${PV}.tar.gz"
```

Comment:
Either add your user to portage group or set DISTDIR to something you can write to.